packages/app/src/app/pages/common/Modals/PreferencesModal/PaymentInfo/index.tsx

```diff
@@ -1,42 +1,52 @@
-import React, { useEffect } from 'react';
+import React, { ComponentProps, FunctionComponent, useEffect } from 'react';
+
 import { SubscribeForm } from 'app/components/SubscribeForm';
 import { useOvermind } from 'app/overmind';
-import { Card } from './Card';
+
 import { Title, Subheading } from '../elements';
-import { Container, CustomSubHeading } from './elements';
 
-export const PaymentInfo: React.FunctionComponent = () => {
+import { Card } from './Card';
+import { Container } from './elements';
+
+export const PaymentInfo: FunctionComponent = () => {
   const {
+    actions: {
+      preferences: { paymentDetailsRequested, paymentDetailsUpdated },
+    },
     state: {
       preferences: {
-        paymentDetailError,
-        paymentDetails: paymentDetailsFromHooks,
         isLoadingPaymentDetails,
+        paymentDetailError,
+        paymentDetails: { brand, last4, name },
```

Comment:
descontring them here will give an error on opening of cannot read brand of undefined